badge-maker/lib/badge-renderers.js

```diff
@@ -802,36 +804,27 @@ function forTheBadge({
   const hasLabel = Boolean(label.length)
   const noText = !hasLabel && !message
   const needsLabelRect = hasLabel || (logo && labelColor)
-  const gutter = noText ? LOGO_TEXT_GUTTER - LOGO_MARGIN : LOGO_TEXT_GUTTER
-  let logoMinX, labelTextMinX
+  let labelTextMinX = TEXT_MARGIN
+  let labelRectWidth = 2 * TEXT_MARGIN
```

Comment:
Fixes issue where we add empty space after the badge when there is no label/logo-in-label

```suggestion
  let labelRectWidth = needsLabelRect ? 2 * TEXT_MARGIN : undefined
```